javaparser-core/src/main/java/com/github/javaparser/ast/nodeTypes/NodeWithOptionalScope.java

```diff
@@ -27,22 +27,11 @@
 import java.util.Optional;
 
 /**
- * Represents a node which can have a scope expression eg. method calls or
- * field accesses (object.method(), object.field).
+ * Represents a node which can have a scope expression eg. method calls (object.method()).
```

Comment:
Maybe we can stress the "can" and say explicitly that this is interface is for the nodes which may or may not have a scope. It should not be used for the nodes that must have a scope